### DISCUSSION TITLE
Switching from WG-Easy to WGDashboard

### DISCUSSION THREAD

**Discussion by qvipin** [WGDashboard/WGDashboard#478] [Q&A]
As someone switching from something overly simplified to this dashboard, I face many troubles. I can get this to work, but I can only access local IPs for some reason, and it doesn't show any traffic on the dashboard when the connection is alive. My settings are all default except that I'm on a lower port of 1000 as people say u want to be below 1024. Help would be nice

**Comment by donaldzou** [comment]
Hi @qvipin, do you mind posting your configuration and see what's wrong with it? (please remove all keys) :)

**Reply by qvipin** [reply]
```
Old config (wg-easy)

[Interface]
PrivateKey = [Redacted]
Address = 10.8.0.4/24
DNS = 1.1.1.1

[Peer]
PublicKey = [Redacted]
PresharedKey = [Redacted]
AllowedIPs = 0.0.0.0/0, ::/0
PersistentKeepalive = 0
Endpoint = [Redacted]:51820
```

```
new config 

[Interface]
PrivateKey = [Redacted]
Address = 10.8.0.1/32
MTU = 1420
DNS = 1.1.1.1

[Peer]
PublicKey = [Redacted]
AllowedIPs = 0.0.0.0/0, ::/0
Endpoint = [Redacted]:51820
PersistentKeepalive = 21
PresharedKey = [Redacted]

```

at some point I got one config to work which was like

```
[Interface]
PrivateKey = [Redacted]
Address = 10.8.0.1/32
MTU = 1420
DNS = 1.1.1.1

[Peer]
PublicKey = [Redacted]
AllowedIPs = 0.0.0.0/0
Endpoint = [Redacted]:1000
PersistentKeepalive = 0
PresharedKey = [Redacted]
```

but that would only work for local ips on the network. now if I even try that config again it refuses to work. I have everything port forwarded, but now im just genuinely confused.

**Reply by qvipin** [reply]
any idea on why this could be happening?

**Reply by donaldzou** [reply]
When you were using `wg-easy`, did you used any PostUp or PreUp commands to setup `iptables`?

Here's an example of my `wg0.conf` (server-side)

```
[Interface]
Address = 10.200.200.1/24
SaveConfig = true
PostUp = iptables -A FORWARD -i wg0 -j ACCEPT; iptables -A FORWARD -o wg0 -j ACCEPT; iptables -t nat -A POSTROUTING -o enp0s1 -j MASQUERADE;
PostDown = iptables -D FORWARD -i wg0 -j ACCEPT; iptables -D FORWARD -o wg0 -j ACCEPT; iptables -t nat -D POSTROUTING -o enp0s1 -j MASQUERADE;
ListenPort = 51820
PrivateKey = 8DsSMli3okgUx5frKbFQ0fMW5ZMyqyxOdOW7+g21L18=

[Peer]
PublicKey = txNOO4ELPHtDRu/zyg+MFdOUlAwMo4i7Ki2n9Pu/mmY=
AllowedIPs = 10.200.200.30/32
```

**Reply by qvipin** [reply]
I only used the default settings as per the installation guides.

**Reply by DaanSelen** [reply]
> ```
> Old config (wg-easy)
> 
> [Interface]
> PrivateKey = [Redacted]
> Address = 10.8.0.4/24
> DNS = 1.1.1.1
> 
> [Peer]
> PublicKey = [Redacted]
> PresharedKey = [Redacted]
> AllowedIPs = 0.0.0.0/0, ::/0
> PersistentKeepalive = 0
> Endpoint = [Redacted]:51820
> ```
> 
> ```
> new config 
> 
> [Interface]
> PrivateKey = [Redacted]
> Address = 10.8.0.1/32
> MTU = 1420
> DNS = 1.1.1.1
> 
> [Peer]
> PublicKey = [Redacted]
> AllowedIPs = 0.0.0.0/0, ::/0
> Endpoint = [Redacted]:51820
> PersistentKeepalive = 21
> PresharedKey = [Redacted]
> ```
> 
> at some point I got one config to work which was like
> 
> ```
> [Interface]
> PrivateKey = [Redacted]
> Address = 10.8.0.1/32
> MTU = 1420
> DNS = 1.1.1.1
> 
> [Peer]
> PublicKey = [Redacted]
> AllowedIPs = 0.0.0.0/0
> Endpoint = [Redacted]:1000
> PersistentKeepalive = 0
> PresharedKey = [Redacted]
> ```
> 
> but that would only work for local ips on the network. now if I even try that config again it refuses to work. I have everything port forwarded, but now im just genuinely confused.

Does your WireGuard say connected and receive and send bytes? Like this:
![image](https://github.com/user-attachments/assets/8b259f20-13ba-4441-805f-acd94c738880)

**Reply by qvipin** [reply]
It sends bytes, but doesnt receive anything.

**Reply by DaanSelen** [reply]
> It sends bytes, but doesnt receive anything.

What are you Iptables rules?

**Comment by kuzzrus** [comment]
I encountered the same problem, only it doesn’t allow me to access the Internet locally.

**Reply by DaanSelen** [reply]
What are your Iptables rules.

**Reply by kuzzrus** [reply]
> Каковы ваши правила Iptables?

I don't quite understand how to fill it out correctly.

**Reply by DaanSelen** [reply]
Can you use the default configuration from LXC or Docker? Do you need help?

**Reply by kuzzrus** [reply]
> Docke

Docker didn't start, it gives an error.
LXС?
Docker didn't start, it gives an error.
LXS?
It would seem that there is nothing complicated, but alas, something went wrong )))
Light WG works, but I want to touch this panel )

**Reply by DaanSelen** [reply]
LXC = Linux Container, a feature of Proxmox. Have you used the: https://donaldzou.dev/WGDashboard-Documentation/install.html

Otherwise perhaps enter the Discord or Element for more live support on this. But here is also possible.

**Reply by kuzzrus** [reply]
> LXC = Linux Container, функция Proxmox. Вы использовали: https://donaldzou.dev/WGDashboard-Documentation/install.html
> 
> В противном случае, возможно, войдите в Discord или Element для более живой поддержки по этому вопросу. Но здесь также возможно.

Yes, if possible, then discord is possible

**Reply by DaanSelen** [reply]
Join the link in the README.md (or here: https://discord.gg/72TwzjeuWm ) and open a thread in #support
